docker-compose.yml

```diff
@@ -35,8 +35,18 @@ services:
     ports:
       - "3000:3000"
       - "1080:1080"
+    environment:
```

Comment:
Why is it also in the dev docker-compose file?

Reply:
So you can develop feature tests. I picked the selenium vnc way because performance does not matter during development but having access to the browser in testing is *very* useful. Especially if you're changing webui a lot.

You add a pry/sleep in your spec and then attach vnc to localhost and can play with chrome in developer mode checking the HTML generated in this very scenario.

I'll gladly give a demo on my workflow

Reply:
But do I want this container always running in development? Or would it make sense that we move it to a dedicated compose file and you would need to start like this:

```
docker-compose -f docker-compose.yml -f docker-compose.chrome.yml up
```

Because most of the time I probably don't need the chrome container running, do I?

Reply:
No, obviously you only need it you want to develop/debug a feature test. If we can make this dynamic, it would be good - if we add chromedriver to the dev container, we can make this completely optional.